{{cookiecutter.project_slug}}/backend/backend.dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM tiangolo/uvicorn-gunicorn-fastapi:python3.7
 
-RUN pip install celery~=4.3 passlib[bcrypt] tenacity requests emails "fastapi>=0.47.0" "uvicorn>=0.11.1" gunicorn pyjwt python-multipart email_validator jinja2 psycopg2-binary alembic SQLAlchemy email_validator
+ADD requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
If doing this, I would prefer to add Poetry to the Docker images and install with Poetry without creating an env inside the Dockerfiles instead of having a pre-build step outside of Docker that depends on Poetry installed locally.

Reply:
Then I'll use multi-build stage in Dockerfile, alright?